src/test/data/com/hotels/bdp/waggledance/manifest/ManifestAttributesTest/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
```

Comment:
This has waggledance in path name, also, don't think we need this in this library?

Reply:
these slipped through